docs/blog/2019-05-29-intergrating-graphiql-explorer/index.md

```diff
@@ -0,0 +1,62 @@
+
+<video controls="controls" autoplay="true" loop="true">
+  <source type="video/mp4" src="/graphiql-explore.mp4"></source>
+  <p>Your browser does not support the video element.</p>
+</video>
+
+## What is GraphiQL Explorer?
+
+[`graphiql-explorer`](https://github.com/OneGraph/graphiql-explorer) is a plugin for GraphiQL that adds a new technique to explore and build GraphQL queries. It adds a graphical representation of available fields and inputs that can be used in queries. It also allows constructing full queries by clicking through available fields and inputs without the repetitive process of typing these queries out by hand!
+
+## Why is GraphiQL Explorer?
+
+We often hear that many developer's first usage of GraphQL is through Gatsby. GraphQL, like any technology, has a learning curve. We can't eliminate this learning curve completely but we can try to make it smoother. As mentioned previously, GraphiQL Explorer allow users to build full GraphQL queries without typing a single line of code. This enables users that don't yet fully understand the GraphQL query syntax to _learn_ GraphQL much more easily:
+
+<video controls="controls" autoplay="true" loop="true">
+  <source type="video/mp4" src="./graphiql-explorer-demo.mp4"></source>
+  <p>Your browser does not support the video element.</p>
+</video>
+
+For more information, check out ["How OneGraph onboards users who are new to GraphQL"](https://www.onegraph.com/blog/2019/01/24/How_OneGraph_onboards_users_new_to_GraphQL.html) blog post for more details about it from [OneGraph](https://www.onegraph.com/) (creators of GraphiQL Explorer).
+
+## Advanced usecases
+
+Improvements to on-boarding users new to GraphQL isn't the only goal of integrating GraphiQL Explorer into Gatsby. There are other pain points that are getting addressed with this addition. Specifically [union types and inline fragments](https://graphql.org/learn/queries/#inline-fragments). If the user is not aware of this syntax to query this type of fields it can be a fairly frustrating experience! GraphiQL Explorer helps solve this problem by listing available types in union:
+
+<video controls="controls" autoplay="true" loop="true">
+  <source type="video/mp4" src="./graphiql-explorer-union-demo.mp4"></source>
+  <p>Your browser does not support the video element.</p>
+</video>
+
+## Try it now
+
+We recently [added](https://github.com/gatsbyjs/gatsby/pull/14280) GraphiQL Explorer to Gatsby. It's available in starting with `gatsby@2.7.3`.
+
+Create new Gatsby project:
+
+```shell
+gatsby new gatsby-with-explorer
+```
+
+or update Gatsby in your existing project:
+
+```shell
+npm install gatsby
```

Comment:
Should this include `@latest` or `@^2.7.3`?

Reply:
this is equivalent to `gatsby@latest`, it will update to latest version